src/test/scala/uk/gov/hmrc/api/helpers/MonthlyReturnsSubmissionHelper.scala

```diff
@@ -14,16 +14,19 @@
  * limitations under the License.
  */
 
-package uk.gov.hmrc.api.utils
+package uk.gov.hmrc.api.helpers
 
-import scala.util.Random
+import play.api.libs.ws.StandaloneWSResponse
+import uk.gov.hmrc.api.service.MonthlyReturnsSubmissionService
 
-object SubPathGenerator {
-  def generateReturnPath(): String = {
-    val randomNumber = generateRandomNumber(1000, 10000)
-    s"$randomNumber/submission/$randomNumber"
-  }
+class MonthlyReturnsSubmissionHelper {
```

Comment:
Same point here - looks like a straight call through to the service. Could remove entirely and just have less code to maintain?